src/AddScores.js

```diff
@@ -0,0 +1,27 @@
+const userName = document.getElementById('userName');
+const userScore = document.getElementById('userScore');
+const error = document.getElementById('error');
+const submitBtn = document.getElementById('submitBtn');
+
+export default function addScore(scoreList) {
+  if (userName.value.trim() === '' || userScore.value.trim() === '') {
+    error.classList.remove('d-none');
+    error.classList.add('d-grid');
+    submitBtn.classList.remove('submit');
+  } else {
+    const score = { name: userName.value, score: userScore.value };
+    scoreList.push(score);
+  }
+}
```

Comment:
- [ ] I would recommend to use arrow functions instead of the `function` keyword. If you check just above your project requirements, you will find `Javascript` requirements. If you open the link and go to point 4, it encourages the use of `ES6`. `Javascript's` `ES6` brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more.

Reply:
I usually use the arrow function, but when using export default with the arrow function it raises an error in the linters. Should I just ignore the error with a comment and use the arrow function then?

Reply:
Does this gives you an error ?

```
export default const addScore = (score..) => {
 ...
}
```

Reply:
Yes. When using the code like this:

``` export default const addScore = (scoreList) => {
  if (userName.value.trim() === '' || userScore.value.trim() === '') {
    error.classList.remove('d-none');
    error.classList.add('d-grid');
    submitBtn.classList.remove('submit');
  } else {
    const score = { name: userName.value, score: userScore.value };
    scoreList.push(score);
  }
}
```

I get this error: 'Parsing error: Only expressions, functions or classes are allowed as the `default` export.'

Reply:
Then you can simply use arrow function and export it later.

```
const addScore = (scoreList) => {
....
}

export default addScore;
```

Reply:
Ok, done.